src/main/java/teammates/ui/webapi/UpdateFeedbackSessionLogsAction.java

```diff
@@ -0,0 +1,63 @@
+package teammates.ui.webapi;
+
+import java.time.Instant;
+import java.time.temporal.ChronoUnit;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import teammates.common.datatransfer.FeedbackSessionLogEntry;
+import teammates.common.exception.EntityAlreadyExistsException;
+import teammates.common.exception.InvalidParametersException;
+import teammates.common.util.Logger;
+import teammates.common.util.TimeHelper;
+import teammates.storage.sqlentity.FeedbackSessionLog;
+
+/**
+ * Process feedback session logs in the past defined time period and store in
+ * the database.
+ */
+public class UpdateFeedbackSessionLogsAction extends AdminOnlyAction {
+
+    static final int COLLECTION_TIME_PERIOD = 60; // represents one hour
+    static final long SPAM_FILTER = 2000L; // in ms
+    private static final Logger log = Logger.getLogger();
+
+    @Override
+    public JsonResult execute() {
+        List<FeedbackSessionLog> filteredLogs = new ArrayList<>();
+
+        Instant endTime = TimeHelper.getInstantNearestHourBefore(Instant.now());
+        Instant startTime = endTime.minus(COLLECTION_TIME_PERIOD, ChronoUnit.MINUTES);
+
+        List<FeedbackSessionLogEntry> logEntries = logsProcessor.getFeedbackSessionLogs(null, null,
+                startTime.toEpochMilli(), endTime.toEpochMilli(), null);
```

Comment:
Correctness of this action relies on the ordering added in this method. Would it be better if there's some indication about needing the ordering? I'm thinking `getOrderedFeedbackSessionLogs`, it's not great, open if there's better suggestions also

Reply:
Hmm thats a good point, I think your suggestion should work fine since there is no downside to having it be ordered and in the future if there is a need to retrieve ordered logs the method can be easily found